src/components/TransactionBlock/PlrDaoStakingTransactionBlock.tsx

```diff
@@ -248,15 +238,15 @@ const PlrDaoStakingTransactionBlock = ({
 
   const fetchAllAccountBalances = async () => {
     try {
-      let data = await getBalanceForAllChains();
-      let accountBalanceWithSupportedChains: AccountBalance[] = data.map((d) =>
-        d.status === 'fulfilled' ? d.value : { chainName: '', keyBasedWallet: 0, smartWallet: 0 }
+      let accountBalanceWithSupportedChains: AccountBalance[] = await Promise.all(
+        supportedChains.map((chain) => getWalletBalance(chain.chainId, chain.title)),
       );
       accountBalanceWithSupportedChains = accountBalanceWithSupportedChains?.filter(
-        (data: AccountBalance) => data.keyBasedWallet > 0 || data.smartWallet
+        (data: AccountBalance) => data.keyBasedWallet > 0 || data.smartWallet > 0,
       );
-      const totalKeyBasedPLRTokens = getTotal(accountBalanceWithSupportedChains as AccountBalance[], 'keyBasedWallet');
-      const totalSmartWalletPLRTokens = getTotal(accountBalanceWithSupportedChains as AccountBalance[], 'smartWallet');
+
+      let totalKeyBasedPLRTokens = getTotal(accountBalanceWithSupportedChains as AccountBalance[], 'keyBasedWallet');
+      let totalSmartWalletPLRTokens = getTotal(accountBalanceWithSupportedChains as AccountBalance[], 'smartWallet');
```

Comment:
why?

Reply:
`const` was as fine, change completely unnecessary and even wrong since both vars will remain the same